app/controllers/close_contacts_controller.rb

```diff
@@ -2,12 +2,37 @@
 
 # CloseContactsController: close contacts
 class CloseContactsController < ApplicationController
+  include CloseContactQueryHelper
+
   before_action :authenticate_user!
   before_action :check_can_create, only: %i[create]
   before_action :check_can_edit, only: %i[update destroy]
   before_action :check_patient
   before_action :check_close_contact, only: %i[update destroy]
 
+  def index
+    redirect_to(root_url) && return unless current_user&.can_view_patient_close_contacts?
```

Comment:
so I was comparing this to the `VaccinesController` and I think we don't need this line here because this check is already handled in `show.html.erb`. we also don't want to be doing any kind of redirecting here because this route/method is used for an AJAX API call

```suggestion
```

Reply:
Good point! Removed.